packages/create-burner/test/mocks/mocks.ts

```diff
@@ -0,0 +1,31 @@
+import { BurnerManager } from "../../src/manager/burnerManager";
+import { Account, RpcProvider } from "starknet";
+import { BurnerConnector } from "../../src/connectors/burner";
+
+export const getBurnerManager = (): BurnerManager => {
+    const burnerManager = new BurnerManager({
+        masterAccount: new Account(
+            new RpcProvider({ nodeUrl: "http://localhost:5050" }),
+            "0x517ececd29116499f4a1b64b094da79ba08dfd54a3edaa316134c41f8160973",
+            "0x1800000000300000180000000000030000000000003006001800006600"
```

Comment:
Can you like these values to the consts in the core package. These are likely to change as Katana evolves, so we need a single source of truth.

Reply:
Done